R/mailR.R

```diff
@@ -186,6 +186,12 @@ send.mail <- function(from, to, subject = "", body = "", encoding = "iso-8859-1"
   if("tls" %in% names(smtp))
     if(smtp$tls)
       email$setTLS(TRUE)
+      
+  if("socketConnectionTimeout" %in% names(smtp))
+    email$setSocketConnectionTimeout(as.integer(smtp$socketConnectionTimeout));
```

Comment:
I don't mind, I just wanted as little impact on existing code as possible and to reflect the structure of the Apache email package.  We could use just one timeout variable.

Charles

Sent from my iPhone

> On 2 Jul 2015, at 21:22, Rahul Premraj <notifications@github.com> wrote:
> 
> In R/mailR.R:
> 
> > @@ -186,6 +186,12 @@ send.mail <- function(from, to, subject = "", body = "", encoding = "iso-8859-1"
> >    if("tls" %in% names(smtp))
> >      if(smtp$tls)
> >        email$setTLS(TRUE)
> > +      
> > +  if("socketConnectionTimeout" %in% names(smtp))
> > +    email$setSocketConnectionTimeout(as.integer(smtp$socketConnectionTimeout));
> I wonder if it would make more sense (and be easier to use) if we instead include timeout as a parameter in send.mail() with a default value. What do you think?
> 
> —
> Reply to this email directly or view it on GitHub.
>